recipes/make/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "4.2.1":
+    url: "http://ftpmirror.gnu.org/gnu/make/make-4.2.1.tar.bz2"
```

Comment:
Is this a legal source? ftpmirror?

Reply:
absolutely: https://www.gnu.org/prep/ftp.html
```
GNU software is available on ftp.gnu.org under the directory /gnu. Please try to use one of the many mirrors of our site listed below: the mirrors will give you faster response. Some mirrors also provide a copy of the rest of ftp.gnu.org; try the parent directory if you're interested. You can use the generic url http://ftpmirror.gnu.org to automatically choose a nearby and up-to-date mirror.

```